packages/mdc-slider/foundation.ts

```diff
@@ -291,76 +292,68 @@ class MDCSliderFoundation extends MDCFoundation {
     this.handlingThumbTargetEvt_ = false;
     this.setActive_(true);
 
-    const moveHandler = (evt) => {
-      this.handleMove_(evt);
+    const moveHandler = (moveEvent: MouseLikeEvent) => {
+      this.handleMove_(moveEvent);
     };
 
+    const moveEventType = MOVE_EVENT_MAP[downEvent.type as DownEventType];
+
     // Note: upHandler is [de]registered on ALL potential pointer-related release event types, since some browsers
     // do not always fire these consistently in pairs.
     // (See https://github.com/material-components/material-components-web/issues/1192)
     const upHandler = () => {
       this.handleUp_();
-      this.adapter_.deregisterBodyInteractionHandler(MOVE_EVENT_MAP[evt.type], moveHandler);
+      this.adapter_.deregisterBodyInteractionHandler(moveEventType, moveHandler as EventListener);
```

Comment:
these EventListeners don't need to be cast after rename to SpecificEventListener

Reply:
Done